var/spack/repos/builtin/packages/visit/package.py

```diff
@@ -38,33 +38,65 @@ class Visit(CMakePackage):
     version('2.10.2', '253de0837a9d69fb689befc98ea4d068')
     version('2.10.1', '3cbca162fdb0249f17c4456605c4211e')
 
+    variant('gui',    default=True, description="Enable VisIt's GUI")
+    variant('hdf5',   default=True, description='Enable HDF5 file format')
+    variant('silo',   default=True, description='Enable Silo file format')
+    variant('python', default=True, description='Enable Python support')
+    variant('parallel', default=False, description='Enable parallel engine')
```

Comment:
we usually call this variant `mpi` as well in spack and make it `default=True`: https://spack.readthedocs.io/en/latest/packaging_guide.html#variant-names